checkov/terraform/checks/resource/ncp/NKSControlPlaneLogging.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class NKSControlPlaneLogging(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure NKS control plane logging enabled for all log types"
+        id = "CKV_NCP_22"
+        supported_resources = ('ncloud_nks_cluster',)
+        categories = (CheckCategories.KUBERNETES,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return 'log/0/audit/0'
+
+    def get_expected_values(self):
+        return [True]
```

Comment:
```suggestion
```
you can remove this, because the default for expected value in our code is `True` 😄